articles/openstack/ostack-sso-rcfile.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How to use the OpenStack API using an SSO enabled user | UKCloud Ltd
+description: Explains the changes needed to make use of the OpenStack API with an SSO enabled user.
+services: openstack
+author: Steve Relf
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenStack API with SSO users
+toc_fullpath: How To/ostack-sso-rcfile.md
+toc_mdlink: ostack-sso-rcfile.md
+---
+
+# How to use the OpenStack API using an SSO enabled user
+
+## Overview
+
+In order to use the OpenStack API with your SSO enabled user, you will need to make a few changes to your downloaded OpenStack RC file. This is to allow your API calls to be correctly passed to the SSO service.
```

Comment:
Suggest:
If your OpenStack user account is SSO-enabled, if you want to use the OpenStack API you'll need to make a few changes to your downloaded OpenStack RC file. These changes allow any API calls that you make to be correctly passed to the SSO service.